Jenkinsfile.yml

```diff
@@ -22,7 +22,7 @@ node('ubuntu18.04-OnDemand'){
    {
 
      // Checking out the smoke test code
-     checkout([$class: 'GitSCM', branches: [[name: '*/master']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'test-fidoiot']], userRemoteConfigs: [[credentialsId: 'sdo-automationgithubtoken', url: 'https://github.com/secure-device-onboard/test-fidoiot']]])
+     checkout([$class: 'GitSCM', branches: [[name: '*/master']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'test-fidoiot']], userRemoteConfigs: [[credentialsId: 'sdo-automationgithubtoken', url: 'https://github.com/fido-device-onboard/test-fidoiot']]])
```

Comment:
We have sdo in credentialsId: 'sdo-automationgithubtoken'.
Whether its ok to keep it as sdo / should we modify it?

Reply:
i think it is ok. we can keep as is for now.